src/main/kotlin/ru/hse/dynamomock/model/TableDef.kt

```diff
@@ -41,5 +44,62 @@ private fun getPartitionKey(keySchema: List<KeySchemaElement>) =
 private fun getSortKey(keySchema: List<KeySchemaElement>) =
     keySchema.firstOrNull { it.isSort }?.attributeName()
 
+private fun checkLocalSecondaryIndexes(indexes: List<LocalSecondaryIndex>, tableKeys: List<KeySchemaElement>): List<LocalSecondaryIndex> {
+    if (indexes.size > 5) {
+        throw DynamoDbException.builder()
+            .message("Cannot have more than 5 local secondary indexes per table")
+            .build()
+    }
+    val indexesNames = mutableSetOf<String>()
+    indexes.forEach {
+        val name = it.indexName()
+        if (indexesNames.contains(name)) {
+            throw DynamoDbException.builder()
+                .message("Two local secondary indices have the same name")
+                .build()
+        }
+        if (name == null || !(name.length in 3..255 && name.matches("[a-zA-Z0-9-_.]+".toRegex()))) {
```

Comment:
I'm not sure it's okay to create and compile a regexp each time. Anyway, it looks like we should move it out somewhere.